### ISSUE TITLE
Missing language tag for alt text: Government of Canada / Gouvernement du Canada

### ISSUE THREAD

**Issue by alexadavidson** [cds-snc/gcds-components#909]
**issue**:
language tag is missing for alternate text, needed for AT

**client feedback**:
> The FIP alternative text “Government of Canada / Gouvernement du Canada” is not tagged with the proper language for the text readers

![Image](https://github.com/user-attachments/assets/caa67773-d43f-4136-91c7-b7b33c1dba7d)

source:
[freshdesk ticket 23117](https://cds-snc.freshdesk.com/a/tickets/23117)
